docs/source/tutorial.md

```diff
@@ -1,24 +1,43 @@
-# Tutorial
+# Tutorials
 
-This page covers brief tutorials and notes on how to use CytoTable.
+Start here if you are new to CytoTable. We’ve split material by audience:
 
-## CellProfiler CSV Output to Parquet
+- **Image analysts (no engineering background required):** follow the narrative tutorials below. They include downloadable data, exact commands, and what to expect.
+- **Engineers / power users:** see the Software Engineering Guide for tuning and integration details, or use the quick recipe below.
 
-[CellProfiler](https://cellprofiler.org/) pipelines or projects may produce various CSV-based compartment output (for example, "Cells.csv", "Cytoplasm.csv", etc.).
-CytoTable converts this data to Parquet from local or object-storage based locations.
+```{admonition} Who this helps (and doesn’t)
+- Helps: image analysts who want to get CellProfiler/NPZ outputs into Parquet with minimal coding; people comfortable running a few commands.
```

Comment:
```suggestion
- Helps: image analysts who want to get CellProfiler/DeepProfiler/InCarta outputs into Parquet with minimal coding; people comfortable running a few commands.
```

Consistent with what you have below. Not sure if you mention anywhere that DeepProfiler outputs as NPZ.